pkg/azuredisk/controllerserver.go

```diff
@@ -104,8 +104,10 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		diskAccessID            string
 		maxShares               int
 		enableBursting          *bool
+		localCloud              *azure.Cloud
 	)
 
+	localCloud = d.cloud
```

Comment:
I believe we also want to do this with `CreateSnapshot` as well since 3rd party backup vendors will want to attribute snapshots to their products' backup workflows.